### ISSUE TITLE
Request for best checkpoint from training

### ISSUE THREAD

**Issue by bzha5848** [open-mmlab/mmdetection#8426]
Hi there,

I'm wondering if there's a way to modify checkpoint hook or make config that let the model updat/store the best pth instead of the lastest?

Thanks so much!!

**Comment by zhangzscn** [comment]
evaluation = dict(
    save_best='auto',
    # The evaluation interval is 'interval' when running epoch is
    # less than ‘max_epochs - num_last_epochs’.
    # The evaluation interval is 1 when running epoch is greater than
    # or equal to ‘max_epochs - num_last_epochs’.
    interval=interval,
    dynamic_intervals=[(max_epochs - num_last_epochs, 1)],
    metric='bbox')

**Comment by bzha5848** [comment]
Hi there,

Thanks for your comments!! I'm also wondering how to fine-tune the pre-trained model in mmdetection?

Best Regards
Bohan
________________________________

**Comment by Czm369** [comment]
Maybe you can load_from a model to fine-tune, refer to https://github.com/open-mmlab/mmdetection/blob/master/docs/en/1_exist_data_model.md#training-on-a-single-gpu

**Comment by JNaranjo-Alcazar** [comment]
I dont get where to add the evaluation dictionary.... On schedule config?
Thanks!

**Comment by Tongfengyu** [comment]
Hi! In mmdet3.0x，some apis changes,and the method you gave above can't work.
evaluation = dict(interval=1, metric='mAP', save_best='auto')
So, how to save best ckpt in mmdet3.0x?

**Comment by Tongfengyu** [comment]
checkpoint=dict(type='CheckpointHook', interval=1, save_best='auto',max_keep_ckpts=3)
It works!
